scopes/component/dev-files/dev-files.docs.mdx

```diff
@@ -0,0 +1,8 @@
+---
+description: Defines Component files of a certain pattern as development files
```

Comment:
Classifies component files for development purposes

Reply:
done